src/lamplib/src/genny/toolchain.py

```diff
@@ -172,6 +191,7 @@ def __init__(
         workspace_root: str,
         os_family: str,
         linux_distro: str,
+        arch: str,
```

Comment:
another nit: can this be `triplet_arch` more pervasively?

Reply:
Done